src/test/java/guitests/IssueSelectedEventTest.java

```diff
@@ -0,0 +1,68 @@
+package guitests;
+
+import javafx.scene.control.ContextMenu;
+import javafx.scene.control.MenuItem;
+import javafx.scene.input.KeyCode;
+import org.junit.Before;
+import org.junit.Test;
+import ui.UI;
+import ui.listpanel.ListPanel;
+import util.events.IssueSelectedEventHandler;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class IssueSelectedEventTest extends UITest{
+
+    static int eventTestCount;
```

Comment:
Instead of doing this (which isn't thread-safe without synchronisation; `++` is not atomic, and the field isn't private), you can use an AtomicInteger and make it local to each test method. There are examples of this in the PlatformEx tests.